Initial 8devices binding commit

mrNedas opened the pull request.

mrNedas committed:
Initial 8devices binding commit

GiedriusM commented on addons/binding/org.openhab.binding.eightdevices/ESH-INF/binding/binding.xml:
```diff
@@ -0,0 +1,11 @@
+				 xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+				 xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+	<name>EightDevices Binding</name>
```
Šituos reiktų tiksliau pamodifikuoti, t.y. kad būtų 8devices ir autoriaus pilnas vardas ir pavardė

GiedriusM commented on addons/binding/org.openhab.binding.eightdevices/ESH-INF/i18n/eightdevices_xx_XX.properties:
```diff
@@ -0,0 +1,13 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```
Šitą failą reikia sutvarkyti pagal reikalavimus: xx_XX pakeisti į en_US ir visus aprašymus sudėti pagal tai kokie yra daiktai ir kanalai

GiedriusM commented on addons/binding/org.openhab.binding.eightdevices/ESH-INF/thing/thing-types.xml:
```diff
@@ -0,0 +1,189 @@
+	
+	
+	<thing-type id="s3700">
+        <label>EightDevices Binding Thing</label>
```
Tekste EightDevices galima pervadinti į 8devices

GiedriusM commented on addons/binding/org.openhab.binding.eightdevices/ESH-INF/thing/thing-types.xml:
```diff
@@ -0,0 +1,189 @@
+	
+	<thing-type id="s3700">
+        <label>EightDevices Binding Thing</label>
+        <description>Sample thing for EightDevices Binding</description>
```
Čia taip pat. Ir iš auto-generated į tikslesnį descriptioną padaryt.

GiedriusM commented on addons/binding/org.openhab.binding.eightdevices/ESH-INF/thing/thing-types.xml:
```diff
@@ -0,0 +1,189 @@
+	
+	
+	<thing-type id="s3800">
+		<label>EightDevices Binding Thing</label>
```
Tuos pačius aprašymus patvarkyt ir lygiavimą sutvarkyt. Galbūt kodo formatavimas gali teisingai išlygiuot.

GiedriusM commented on addons/binding/org.openhab.binding.eightdevices/ESH-INF/thing/thing-types.xml:
```diff
@@ -0,0 +1,189 @@
+        </config-description>
+	</thing-type>
+
+	<!-- Sample Channel Type -->
```
Šia auto-generated kanalą išimti

mrNedas committed:
*Added more detailed descriptions, labels.
*Changed properties file name (xx_XX to en_US)
*Added descriptions and labels of all thing types, configurations and channel types.
*Formatted thing-types.xml file
